imports/plugins/included/jobcontrol/server/jobs/cleanup.js

```diff
@@ -0,0 +1,47 @@
+import { Jobs } from "/lib/collections";
+import { Hooks, Logger } from "/server/api";
+
+if (Hooks) {
```

Comment:
Now that we have module imports, we don't really need to check for the `Hooks` global anymore.  If that was null, the import at the top of this file would throw an error before you even get to that check.